common/api/endpoint/ApiEndPoint.ts

```diff
@@ -7,6 +7,7 @@ export const ApiEndPoint = {
   },
   // TODO: Remove deprecated routes (still in use in _legacy)
   Auth: {
+    authorize: () => apiPath('authorize'),
```

Comment:
can't we get the user on the init endpoint ?

Reply:
We need separate endpoint for authorization
We can also return the user on init

We need later on handle token refresh